packages/ui/src/components/ListV2/index.tsx

```diff
@@ -139,5 +139,5 @@ List.Cell = ListCell
 List.Headers = ListHeaders
 List.HeaderRow = ListHeaderRow
 List.Header = ListHeader
-List.Placeholder = ListLoadingPlaceholder
+List.Skeleton = ListLoadingSkeleton
```

Comment:
That's a breaking change for ListV2 we need to take in account when making change on every projects